addons/getLoadoutsFromUnits/functions/fnc_getItems.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+params ["_items", "_tab"];
+diag_log format ["Items: %1", _items];
+private _return = [];
+private _count = count _items;
+
+diag_log format ["Items count: %1", _count];
+
+if (_count > 0) then {
+   private _quadrupleTab = _tab + _tab + _tab + _tab;
+   {
+       if (count _x == 3) then {
+          _return pushBack (_quadrupleTab + format ["LIST_%1('%2')", _x select 2, _x select 0]);
```

Comment:
List macro only goes to 30.

Reply:
And?

Reply:
I don't know, are you handling that? What happens if a dude has 31 items of a kind in his backpack?